js/start.js

```diff
@@ -305,44 +346,63 @@ function fetchStartupData() {
           walletBalanceFetchFailed = true;
         } else if (jqXhr === searchProvidersFetch) {
           searchProvidersFetchFailed = true;
+        } else if (jqXhr === verifiedModsFetch) {
+          verifiedModsFetchFailed = true;
         }
       }
     })
     .done(() => {
       fetchStartupDataDeferred.resolve();
+      setInterval(() => fetchVerifiedMods(), 360000);
     })
     .fail((jqXhr) => {
       const curConn = getCurrentConnection();
 
-      if (!jqXhr || !curConn || curConn.status !== 'connected') {
+      // Check to see if the server is not connected. the search providers and verified moderators
+      // can return an undefined jqXhr if their external endpoints are down while the app itself is
+      // still functional.
+      if ((ownFollowingFailed || walletBalanceFetchFailed) &&
+        (!jqXhr || !curConn || curConn.status !== 'connected')) {
         // the connection management modal should be up with relevant info
         console.error('The startup data fetches failed. Looks like the connection to the ' +
           'server was lost.');
         return;
       }
 
-      if (ownFollowingFailed || walletBalanceFetchFailed || searchProvidersFetchFailed) {
+
+      if (ownFollowingFailed || walletBalanceFetchFailed || searchProvidersFetchFailed ||
+        verifiedModsFetchFailed) {
         let title = '';
+        let msg = jqXhr && jqXhr.responseJSON && jqXhr.responseJSON.reason || '404';
+        let btnText = app.polyglot.t('startUp.dialogs.btnManageConnections');
+        let btnFrag = 'manageConnections';
```

Comment:
The search and verified fail states should have a continue button instead of a manage connections button, since neither are dependent on the connection to the node.